lib/memory/index.js

```diff
@@ -58,13 +58,13 @@ module.exports = function MemoryStore(data_set) {
 
   const update = R.curry((bucket, id, data) =>
 
-      getById(bucket, id)
+    getById(bucket, id)
 
     .tap(R.when(R.isNil, () => throwItemNotFound(bucket, id)))
 
     .then(() => { store[bucket][id] = data })
 
-    .return(id)
+    .return(1)
```

Comment:
You could just update the kuss wrapper. Since pimp-my-sql is the low-level lib it's probably ok to have it return the affected row count.